analysis/kb_aggregate.py

```diff
@@ -0,0 +1,80 @@
+"""
+Module to create aggregated statistics - per file format, per period - for raw record data acquired from
+KB
+"""
+import datetime
+import json
+import logging
+import os.path
+from argparse import ArgumentParser
+from typing import List
+
+from tqdm import tqdm
+
+from analysis.config import load_config, Config
+
+
+def main(config: Config) -> int:
+    start = datetime.datetime.now()
+
+    kb_cfg = config['data']['kb']
+    file_temp_stats: dict = {}
+    skipped_records = 0
+
+    with open(kb_cfg['raw_csv_path'], 'rt') as f:
+        for line_no, record in enumerate(tqdm(f, total=kb_cfg['raw_csv_line_count'])):
+            # Skip header with column names
+            if line_no == 0:
+                # header = record.split(',')
+                continue
+
+            parts = record.split('|')
+
+            mimetype = str(parts[1])
+            create_date = parts[0]
+
+            file_temp_stats.setdefault(mimetype, {})
+
+            year_month = '-'.join(create_date.split('-')[1:3])
```

Comment:
Aha ik zie al wel iets: de year_month wordt verwacht in YYYY-mm format
```suggestion
            year_month = '-'.join(reversed(create_date.split('-')[1:3]))
```